src/js/dp/feedback-form.js

```diff
@@ -0,0 +1,122 @@
+document.addEventListener("DOMContentLoaded", function () {
+    const pageURL = window.location.href;
+    const feedbackURL = "/feedback";
+
+
+    const feedbackFormContainer = document.querySelector(
+        "#feedback-form-page-container"
+    );
+    if (feedbackFormContainer) {
+        feedbackFormContainer.addEventListener("submit", function (e) {
+            e.preventDefault();
+            const fieldErrors = document.querySelectorAll(
+                "#feedback-form-page-container .feedback-form-control__error"
+            );
+            fieldErrors.forEach((fieldError) => {
+                fieldError.classList.remove("feedback-form-control__error");
+            });
+
+            const formErrors = document.querySelectorAll(
+                "#feedback-form-page-container .feedback-form-error"
+            );
+            formErrors.forEach((formError) => {
+                formError.remove();
+            });
+
+            const emailField = document.querySelector("#email-field");
+            const descriptionField = document.querySelector("#description-field");
```

Comment:
if you use the forms api, this could be something like 
```suggestion
      const formData = new FormData(e.target);
      const emailField = formData.get("#email-field");
      const descriptionField = formData.get("#description-field");
```